doc/poo.md

```diff
@@ -343,13 +343,23 @@ In that list:
     You can specify the empty list `[]` if there are no such super-prototypes.
 
   * The optional following list of parameters are as many of slot names that
-    will be bound to macros to access the relevant slots of the instance being
+    will be bound with macros to access the relevant slots of the instance being
     computed when the particular slot value is computed.
     Definitions for these slots are presumably to be provided by
     other prototypes in the object being instantiated,
     since those in the current prototype will already be implicitly included
     in the list of symbols to bind.
 
+    As an example, we have a `slot-name` x, which is bound in the macro `make-o-with`:
+    ```lisp
+    (def o {(:: @ [] x)
+      x: x}) ;; what is x?
```

Comment:
This `x: x` is a circular reference that will blow the stack and eat all your memory if accessed.

Reply:
And you don't need the first `x`, because it's implicit already from the presence of the `x:` slot.